package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@exabyte-io/application-flavors.js",
+  "version": "2021.04.12",
+  "description": "Computational application configuration for the Exabyte.io platform",
```

Comment:
Input file templates and related computational configuration parameters for the applications used in the platform.

Reply:
addressed in https://github.com/Exabyte-io/exabyte-application-flavors/pull/1/commits/3f32cbed80179642437e6a7140b80a59a0eb664a